examples/express-all-interactions/.gitignore

```diff
@@ -0,0 +1,46 @@
+# Logs
```

Comment:
the top-level `.gitignore` should already be ignoring all the necessary files. for example, `node_modules` in that file works recursively, so it would ignore `node_modules` in this directory.

the only advantage i can see to committing an additional `.gitignore` here would be so people can copy the directory out and use it as an independent application. i don't feel strongly about that, since i think developers should choose their own version control system and set it up how they like. i can be convinced otherwise.

if we do want to add a `.gitignore` here, i would suggest that it be pretty bare. there are a lot of opinions expressed when you take boilerplate like this, and a lot more to maintain. i'd like to reduce it to just the minimum.